assets/js/blocks/attribute-filter/edit.tsx

```diff
@@ -309,7 +308,7 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak }: EditProps ) => {
 					</ToggleGroupControl>
 					<ToggleControl
 						label={ __(
-							'Filter button',
+							"Show 'Apply filters' button",
```

Comment:
I notice that this label is different between blocks in the design:
- `'Apply filters' button`: the Stock filter and Attribute filter blocks.
- `Show 'Apply filters' button`: the Price filter and Rating filter blocks.

If we haven't, I think we should confirm with @vivialice to use only one label across filter blocks.

Reply:
Yep, I used "Show 'Apply filters' button" as its consistent with your Filter by Price component and a more descriptive label. @vivialice would you agree?

Reply:
@tjcafferkey Yes I agree, thank you.